### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by SUPERCILEX**
The cleanup work that was pulled out of #1017

### CODE REVIEW

File: auth/src/main/java/com/firebase/ui/auth/util/PlayServicesHelper.java [hunk 1]
```diff
@@ -11,13 +11,9 @@
  * client application
  */
 public class PlayServicesHelper {
-    private static GoogleApiAvailability mApiAvailability;
-
+    @Deprecated
```
Comment (samtstern) [inline auth/src/main/java/com/firebase/ui/auth/util/PlayServicesHelper.java]: Any reason for this change?  Why not preserve the availability object?

File: auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java [hunk 2]
```diff
@@ -41,7 +41,14 @@ public PreambleHandler(Context context, FlowParameters parameters, @StringRes in
         setupCreateAccountPreamble();
```
Comment (samtstern) [inline auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java]: Should we move this call out of the constructor and into the static `setup` method as well?

File: auth/src/main/java/com/firebase/ui/auth/util/PlayServicesHelper.java (hunk 1 above)
Comment (SUPERCILEX) [reply auth/src/main/java/com/firebase/ui/auth/util/PlayServicesHelper.java]: Isn't it kinda pointless to store a static field in a static field? 😆 I would guess this is old code from before you could get the availability singleton like so: `GoogleApiAvailability.getInstance()`.

Is there any reason to store the singleton's reference in our class?

File: auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java (hunk 2 above)
Comment (SUPERCILEX) [reply auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java]: Good idea, done.

File: auth/src/main/java/com/firebase/ui/auth/util/PlayServicesHelper.java (hunk 1 above)
Comment (samtstern) [reply auth/src/main/java/com/firebase/ui/auth/util/PlayServicesHelper.java]: I see, I forgot that the underlying object was a singleton as well.  :+1:

Comment (SUPERCILEX) [reply auth/src/main/java/com/firebase/ui/auth/util/PlayServicesHelper.java]: Haha, cool beans.

**Pull request merged**